rtl/ecltpl/prototpl.cpp

```diff
@@ -1,21 +0,0 @@
-$?doNotIncludeInGeneratedCode$
-/*##############################################################################
-#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
-#
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
-############################################################################## */
-
-$?$@include@
-@prototype@
-@literal@
-@declare@
```

Comment:
I don't see this file replaced with an embedded string.  Is this one no longer required?

Reply:
correct, this wasn't used anywhere.